static/app/utils/profiling/differentialFlamegraph.tsx

```diff
@@ -0,0 +1,90 @@
+import {FlamegraphTheme} from './flamegraph/FlamegraphTheme';
+import {relativeChange} from './units/units';
+import {Flamegraph} from './flamegraph';
+import {FlamegraphFrame} from './flamegraphFrame';
+
+function countFrameOccurences(frames: FlamegraphFrame[]): Map<string, number> {
+  const counts = new Map<string, number>();
+
+  for (const frame of frames) {
+    const key = frame.frame.name + (frame.frame.file ? frame.frame.file : '');
```

Comment:
Should this be a method on `Frame`?

Reply:
@Zylphrex I thought about it before, but I haven't done it. The reason for that is that this is more the key that we use for color encoding which is not something that the Frame has to know or care about. Maybe a utility fn would be a good middle ground here, wdyt?